egs2/fsc/asr1/run.sh

```diff
@@ -9,7 +9,7 @@ train_set="train"
 valid_set="valid"
 test_sets="test valid"
 
-asr_config=conf/train_asr.yaml
+asr_config=conf/tuning/train_asr_transformer_adam_specaug.yaml
```

Comment:
If this config scores the best, you can change a symlink of `conf/train_asr.yaml` to specify `conf/tuning/train_asr_transformer_adam_specaug.yaml`, and make it as default.

Reply:
Good point! I have made that change.